src/components/PrimaryPanes/Sources.css

```diff
@@ -74,8 +74,9 @@
   display: flex;
 }
 
-.sources-list .managed-tree .tree-node:first-child {
-  margin-top: 4px;
+.sources-list .managed-tree .tree {
+  overflow: visible;
```

Comment:
do we need this? It seems to work fine without it.

Reply:
Nope! I've removed it. Thanks.